cove_iati/rulesets/iati_orgids_ruleset/steps/orgids_step_definitions.py

```diff
@@ -0,0 +1,25 @@
+import re
+
+from behave import then
+
+from cove.lib.common import get_orgids_prefixes
+from cove_iati.rulesets.utils import get_child_full_xpath, get_xobjects, register_ruleset_errors
+
+ORGIDS_PREFIXES = get_orgids_prefixes()
+
+
+@then('`{attribute}` id attribute must start with an org-ids prefix')
+@register_ruleset_errors()
+def step_openag_org_id_prefix_expected(context, attribute):
+    errors = []
+    fail_msg = '@{} {} does not start with a recognised org-ids prefix'
+
+    for xpath in get_xobjects(context.xml, context.xpath_expression):
+        attr_id = xpath.attrib.get(attribute, '')
+        for prefix in ORGIDS_PREFIXES:
+            if re.match('^%s' % prefix, attr_id):
```

Comment:
Since prefixes aren't regexes, would `attr_id.startswith(prefix)` be clearer here?

Reply:
Yes, I just forgot that ```startswith``` exists